Makefile.include

```diff
@@ -141,9 +141,8 @@ ifeq ($(DEVELHELP),1)
   CFLAGS += -DDEVELHELP
 endif
 
-# If not already set in CFLAGS, set logging level by default to INFO, e.g. 3.
-LOG_LEVEL ?= 3
-ifeq (,$(filter -DLOG_LEVEL=%,$(CFLAGS)))
+# Override LOG_LEVEL if variable is set
+ifdef LOG_LEVEL
```

Comment:
Now you don't check anymore if the CFLAGS are already setting `LOG_LEVEL`.

Reply:
That's a choice, if the user already sets the CFLAGS in the Makegile, he is supposed to know what he is doing.

Reply:
How are they supposed to know that?

Reply:
Thinking of it a second time, not checking if CFLAGS is already set would lead to build issue because of LOG_LEVEL macro already defined. I updated this another time.

Reply:
@miri64, are you ok with the latest changes here ?

Reply:
Yes, please squash.